client/tasks/fills/appearance.js

```diff
@@ -137,8 +137,14 @@ class Appearance extends Component {
 				this.setState( { isPending: false } );
 				this.completeStep();
 			} )
-			.catch( ( error ) => {
-				createNotice( 'error', error.message );
```

Comment:
Another option is to sanitize the HTML or strip tags.  We could strip pretty by creating a div element and grabbing the text content.  A utility function that does this might prove useful.